language/predefined/closure/call.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 9c74079f12d67cabb52c124d761f48275417d7eb Maintainer: samesch Status: ready -->
+<!-- Reviewed: no -->
+<refentry xml:id="closure.call" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>Closure::call</refname>
+  <refpurpose>Bindet eine Closure und ruft sie auf</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description"><!-- {{{ -->
+  &reftitle.description;
+  <methodsynopsis role="Closure">
+   <modifier>public</modifier> <type>mixed</type><methodname>Closure::call</methodname>
+   <methodparam><type>object</type><parameter>newThis</parameter></methodparam>
+   <methodparam rep="repeat"><type>mixed</type><parameter>args</parameter></methodparam>
+  </methodsynopsis>
+  <para>
+   Bindet eine Closure vorübergehend an <parameter>newThis</parameter> und
+   ruft sie mit den angegebenen Parametern auf.
+  </para>
+ </refsect1><!-- }}} -->
+
+ <refsect1 role="parameters"><!-- {{{ -->
+  &reftitle.parameters;
+  <variablelist>
+   <varlistentry>
+    <term><parameter>newThis</parameter></term>
+    <listitem>
+     <para>
+      Das <type>Objekt</type>, an das die Closure für die Dauer des Aufrufs
+      gebunden wird.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><parameter>args</parameter></term>
+    <listitem>
+     <para>
+      Null oder mehr Parameter, die der Closure als Parameter übergeben werden.
```

Comment:
Unsure about this: 

```suggestion
      &null; oder mehr Parameter, die der Closure als Parameter übergeben werden.
```

Reply:
In EN it's "zero"

Reply:
Obviously. I must have read the context wrong. 

However, I would try not to translate zero as zero, because that can quickly lead to misunderstandings. How about: 

```suggestion
      Keine oder mehr Parameter, die der Closure als Parameter übergeben werden.
```

Reply:
yes, that's good, but IMHO it should rather be:
```suggestion
      Keinen oder mehr Parameter, die der Closure als Parameter übergeben werden.
```

Reply:
I am unsure about this. My sense of language is that both are possible in some way, but I **personally** think I would prefer my version. Especially as it relates to the possible plural.

Reply:
ACK
I still don't really like any of these alternatives. The japanese translation reads something like this:
```suggestion
      Hier können die Parameter angegeben werden, die an die Closure übergeben werden sollen.
```

Reply:
I like the `optional`. Maybe with `beliebig viele`? Or is that too duplicated? But yes, I am fine with both of the translations! :) 

```suggestion
      Optional können beliebig viele Parameter angegeben werden, die an die Closure übergeben werden sollen.
```